source/game/field/obj/ObjectWoodbox.hh

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "game/field/obj/ObjectBreakable.hh"
+
+namespace Field {
+
+class ObjectWoodbox : public ObjectBreakable {
+public:
+    ObjectWoodbox(const System::MapdataGeoObj &params);
+    ~ObjectWoodbox() override;
+
+    void init() override;
+
+    /// @addr{0x8077ED7C}
+    [[nodiscard]] const char *getKclName() const override {
+        return "ironbox"; // woodbox when not in TTs
+    }
+
+    void calcCollisionTransform() override;
+
+private:
+    static constexpr f32 HALF_SIZE = 100.0f;
```

Comment:
Since `HALF_SIZE` is only used in one location, can we move it to `ObjectWoodbox::calcCollisionTransform()`? Remove the `static` when you do that too